docker/zu/gradlew

```diff
@@ -159,14 +175,9 @@ save () {
     for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ; done
     echo " "
 }
-APP_ARGS=$(save "$@")
+APP_ARGS=`save "$@"`
 
 # Collect all arguments for the java command, following the shell quoting and substitution rules
 eval set -- $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS "\"-Dorg.gradle.appname=$APP_BASE_NAME\"" -classpath "\"$CLASSPATH\"" org.gradle.wrapper.GradleWrapperMain "$APP_ARGS"
 
-# by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong
```

Comment:
why are we removing these?

Reply:
this is auto-generated by `gradle wrapper` and came as part of upgrading gradle from 4.8.1 to 6.3 in zu sub-project
See: https://github.com/pravega/zookeeper-operator/pull/175/files#diff-efc13f2a48bac60893fac008f48da0f7R3